advanced/Scripts/blacklist.sh

```diff
@@ -53,8 +50,8 @@ HandleOther(){
 
 PopBlacklistFile(){
 	#check blacklist file exists, and if not, create it
-	if [[ ! -f ${blacklist} ]];then
-  	  touch ${blacklist}
+	if [[ ! -f ${blacklistFile} ]];then
```

Comment:
Yeah, you're cleaning up a lot of our messes that we needed to get to, so thank you thank you for that!!